1_fetch.yml

```diff
@@ -67,3 +69,8 @@ targets:
   focus_geoms:
     command: fetch_geoms(focus_geoms_config, within = view_polygon)
 
+  1_fetch/out/precip_spatial.rds.ind:
+    command: fetch_precip_spatial(ind_file=target_name,
+                                view_config = view_config)
```

Comment:
I think you want `view_polygon`, not `view_config`

Reply:
👍 will give that a go -- don't think it existed when I started.